app/styles/input.css

```diff
@@ -96,6 +96,7 @@ input[type=checkbox] {
   width: 13px;
   height: 13px;
   vertical-align: bottom;
+  transition: 0.2s background-color linear;
```

Comment:
Other inputs? :)

Reply:
Let's save that for later. The :hover, and selected states use `background-image: linear-gradient()` which seems tricky to animate:

https://stackoverflow.com/questions/6542212/use-css3-transitions-with-gradient-backgrounds/63848864#63848864